content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,245 @@
+
+<div className="tw-mb-sm">
+  <Figure
+    src="https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Blame.png"
+    alt="Git Blame"
+    caption="Git-extras is now part of the platform and core functionality"
+  />
+</div>
+
+<br />
+<Badge link="/code-search" text="Code Search" color="cerise" size="small" />
+
+#### Find answers faster with a revamped reference panel 
+Your code navigation experience just got better with a refreshed version of the reference panel. Stay within your workflow with a new preview panel that helps you [navigate references](https://docs.sourcegraph.com/code_navigation/explanations/features#find-references) without having to leave the file. The consolidation of definitions, references, and implementations in the same tab reduces the number of clicks needed to get pertinent information. You can find the references you're looking for much faster using new filters, so you'll no longer need to scroll through dozens of results.
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Reference%20panel',
+    mp4: 'blog/release-post/4.0/Reference%20panel'
+  }}
+  loop={true}
+  title="Reference panel refresh"
+  caption="Stay within your workflow and quickly access references with the revamped reference panel"
+/>
+
+<a href="https://docs.sourcegraph.com/code_navigation/explanations/features#find-references" className="tw-not-italic tw-flex tw-items-center">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+
+
+### 🏗️ High-leverage ways to improve your entire codebase
+<br />
+<Badge link="/batch-changes" text="Batch Changes" color="blue" size="small" />
+
+#### Make changes across your codebase at enterprise scale with server-side Batch Changes (beta)
+[Batch Changes](https://about.sourcegraph.com/batch-changes) allows you to automate code changes across your entire codebase. Previously, you had to run `src-cli` locally to create batch changes, which could take an impractical amount of time or be brittle for large or resource-intensive batch changes. Now, with server-side Batch Changes, large-scale changes can be run across thousands of repositories with a better development experience. With server-side Batch Changes:
+
+- Run large-scale or resource-intensive batch changes without clogging your local machine.
+- Run large batch changes quickly by distributing them across an autoscaled pool of compute instances.
+- Get a better debugging experience by streaming logs directly into Sourcegraph.
+
+This feature requires admins to set up executors (much like CI agents), which Sourcegraph uses for running large-scale changes. Executors can also be used to run [code navigation auto-indexing](https://github.com/sourcegraph/about/blob/bf341aafde75450c295aaec064470017020cdcd8/content/blogposts/2022/sourcegraph-4.0.md#TODO-link-to-section-or-docs). All Sourcegraph Cloud instances have one or more executors available by default. 
```

Comment:
@malomarrec Updated this from saying " This feature requires admins to set up executors (much like CI agents), which Sourcegraph will use to offload expensive tasks" to "This feature requires admins to set up executors (much like CI agents), which Sourcegraph uses for running large-scale changes." 

Are you good with that change?